packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/queries.ts

```diff
@@ -66,7 +66,7 @@ pools_delegates AS (
             LIMIT 1
       )
    JOIN stake_address sa ON 
-     sa.hash_raw  = pu.reward_addr 
+     sa.hash_raw  = pu.reward_addr_id 
```

Comment:
These are not valid joins. `pu.reward_addr_id` is referencing the `sa.id` (`addr_id`) column